adapters/IronSource/IronSourceAdapter/GADMAdapterIronSourceBannerDelegate.h

```diff
@@ -0,0 +1,30 @@
+//
+//  GADMAdapterIronSourceBannerDelegate.h
+//  Adapter
+//
+//  Created by alond on 30/08/2022.
+//  Copyright © 2022 Google. All rights reserved.
+//
+
+@protocol GADMAdapterIronSourceBannerDelegate
```

Comment:
Why do we need this banner delegate class apart from `ISDemandOnlyBannerDelegate`?